src/refinement-crumbs/index.ts

```diff
@@ -6,19 +6,17 @@ class RefinementCrumbs {
   previousField: string;
 
   init() {
-    let selectedRefinementsUpdated;
-    let navigationSelector;
     switch (this.props.storeSection) {
       case StoreSections.PAST_PURCHASES:
-        selectedRefinementsUpdated = Events.PAST_PURCHASE_SELECTED_REFINEMENTS_UPDATED;
-        navigationSelector = (field) => this.select(Selectors.pastPurchaseNavigation, field);
+        this.state.selectedRefinementsUpdated = Events.PAST_PURCHASE_SELECTED_REFINEMENTS_UPDATED;
```

Comment:
As long as we're sure that `state` is an object at this point, this is fine. I think that's the case.

Reply:
yes https://github.com/groupby/storefront-core/blob/master/src/core/tag/index.ts#L22